src/components/settings/SettingsLMSTab/ExistingCard.jsx

```diff
@@ -50,7 +54,12 @@ const ExistingCard = ({
       setErrorModalText(errorToggleModalText);
       openError();
     } else {
-      onClick(TOGGLE_SUCCESS_LABEL);
+      // eslint-disable-next-line no-lonely-if
+      if (toggle) {
+        onClick(ACTIVATE_TOAST_MESSAGE);
+      } else {
+        onClick(INACTIVATE_TOAST_MESSAGE);
+      }
```

Comment:
nit: A bit terser option
```suggestion
      onClick(toggle ? ACTIVATE_TOAST_MESSAGE : INACTIVATE_TOAST_MESSAGE);
```